doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -73,98 +74,94 @@ sdk\<service name>\<package name>\<package name>.sln
 
   Make sure `npx cadl compile .` runs correctly.
 
-  ***Add cadl-csharp emitter***
+  **Note**: It is recommended to configure Cadl Specs on [REST API specifications](https://github.com/Azure/azure-rest-api-specs). Please refer to the [Guidelines](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/cadl-structure-guidelines.md).
 
-  Modify package.json, add one line under dependencies:
-
-```diff
-        "dependencies": {
-          "@cadl-lang/compiler": "^0.37.0",
-          "@cadl-lang/rest": "^0.19.0",
-          "@azure-tools/cadl-azure-core": "^0.9.0",
-+         "@azure-tools/cadl-csharp": "0.1.8"
-        },
-```
-
-        Run `npm install` again to install @azure-tools/cadl-csharp.
-
-  **Notes**: @azure-tools/cadl-csharp: "0.1.8" only works with @cadl-lang/compiler: "0.37.0" and @cadl-lang/rest: "0.19.0"
-  
-
-  ***Modify (or create) cadl-project.yaml, add one line under emitters:***
-
-  ```diff
-  emitters:
-+  "@azure-tools/cadl-csharp": true
-  ```
-
-  ***Configuration (optional)***
+  ***Configuration***
 
   One can further configure the SDK generated, using the emitter options on @azure-tools/cadl-csharp.
 
-```
-emitters:
+  Modify (or create) cadl-project.yaml, add emitter options under options/@azure-tools/cadl-csharp. Option `namespace` and `emitter-output-dir` are required.
+
+```diff
+parameters:
+  "python-sdk-folder":
```

Comment:
Can we focus one csharp only?  Not sure we should include the settings for other emitters.

Reply:
We recommend the customer to create the cadl project which complying the structure in the spec repo which will require to add those information in the cadl-project.yaml file.

Reply:
We should not be documenting what the spec repo should document.  By the time they get to our document they already have all of this setup.